app/models/issuer.py

```diff
@@ -66,3 +66,18 @@ class RevokeCredential(BaseModel):
     credential_exchange_id: str
     credential_definition_id: Optional[str] = None
     auto_publish_on_ledger: Optional[bool] = False
+
+
+class PublishRevocations(BaseModel):
+    revocationRegistryCredentialMap: Dict[str, Optional[List[str]]] = Field(
+        description="A map of revocation registry ids to credential exchange ids. \
+        The credential exchange ids are optional if you want to publish all for a given revocation registry id.",
+    )
+
+
+class ClearPendingRevocationsRequest(BaseModel):
+    purge: Optional[Dict[str, List[str]]] = Field(
```

Comment:
1) `revocationRegistryCredentialMap` -> `revocation_registry_credential_map` (snake case is the standard in python)

2) Hierdie twee models is soortgelyk. Ek dink die variable naam `revocation_registry_credential_map` moet in albei gebruik word. Want `purge` is regnies ook 'n revocation_registry_credential_map.

3) Die eerste een het 'n `Optional[List[str]` vir die values. Dit kan ook net 'n List[str] wees, waar 'n empty list aandui om alles te publish.

4) `purge` vat tans 'n `Optional[Dict[..]]` Ek dink ons kan Optional hier avoid want as iemand 'n `{}` post dan is dit tegnis 'n Dict met geen keys. So 'n Dict[str, List[str]] is ook fine, en dan kan albei descriptions net aandui dat.

Ek het met ChatGPT se hulp die descriptions refine met hierdie voorstelle:
```py
class PublishRevocations(BaseModel):
    revocation_registry_credential_map: Dict[str, List[str]] = Field(
        default={},
        description="A map of revocation registry IDs to lists of credential exchange IDs that should be published. "
                    "Providing an empty list for a registry ID publishes all pending revocations for that ID. "
                    "An empty dictionary signifies that the action should be applied to all pending revocations across all registry IDs."
    )

class ClearPendingRevocationsRequest(BaseModel):
    revocation_registry_credential_map: Dict[str, List[str]] = Field(
        default={},
        description="A map of revocation registry IDs to lists of credential exchange IDs for which pending revocations should be cleared. "
                    "Providing an empty list for a registry ID clears all pending revocations for that ID. "
                    "An empty dictionary signifies that the action should be applied to clear all pending revocations across all registry IDs."
    )
```